src/StatementParsers/TransactionParser.php

```diff
@@ -33,19 +34,33 @@ public function parse(array $lines): Transaction
 		$valutaDate = new DateTime("0001-01-01");
 		$amount = 0.0;
 		$sepaDirectDebit = null;
+
+        /** @var string|null $transactionSequence */
+        $statementSequence = null;
+
+		/** @var string|null $transactionSequence */
+        $transactionSequence = null;
+
+        /** @var string|null $transactionSequence */
+        $transactionSequenceDetail = null;
```

Comment:
Sequence number detail will always be 0 as this is just a sequence number for a transaction which spans multiple lines. It would be better to just remove this value.